rosdep/python.yaml

```diff
@@ -3501,30 +3501,34 @@ python-rospkg:
     pip:
       packages: [rospkg]
   ubuntu:
-    artful: [python-rospkg]
-    bionic: [python-rospkg]
-    lucid: [python-rospkg]
-    maverick: [python-rospkg]
-    natty: [python-rospkg]
-    oneiric: [python-rospkg]
-    precise: [python-rospkg]
-    quantal: [python-rospkg]
-    raring: [python-rospkg]
-    saucy: [python-rospkg]
-    trusty: [python-rospkg]
+    '*': [python-rospkg]
     trusty_python3: [python3-rospkg]
-    utopic: [python-rospkg]
     utopic_python3: [python3-rospkg]
-    vivid: [python-rospkg]
     vivid_python3: [python3-rospkg]
-    wily: [python-rospkg]
     wily_python3: [python3-rospkg]
-    xenial: [python-rospkg]
     xenial_python3: [python3-rospkg]
-    yakkety: [python-rospkg]
     yakkety_python3: [python3-rospkg]
-    zesty: [python-rospkg]
     zesty_python3: [python3-rospkg]
+python-rospkg-modules:
+  alpine:
+    pip:
+      packages: [rospkg]
+  arch: [python2-rospkg]
+  debian: [python-rospkg-modules]
```

Comment:
I doubt this is a valid Debian package name?

Same below.

Reply:
Why not? We package it for debian ourselves.

From: http://packages.ros.org/ros-shadow-fixed/ubuntu/dists/stretch/main/binary-amd64/Packages
```
Package: python-rospkg-modules
Priority: optional
Section: python
Installed-Size: 130
Maintainer: Ken Conley <kwc@willowgarage.com>
Architecture: all
Version: 1.1.9-1
Replaces: python-rospkg (<< 1.1.0)
Depends: python-yaml, python:any (<< 2.8), python:any (>= 2.7.5-5~), python-catkin-pkg-modules, lsb-release
Conflicts: python-rospkg (<< 1.1.0)
Filename: pool/main/p/python-rospkg-modules/python-rospkg-modules_1.1.9-1_all.deb
SHA512: b6230a36265fa24c0f7d01ad2caadef2a2b32c9ea1a11364a164552b477d550d595cb76bad4654aa2610c774a482c0f190c75e3ef78622ecc71d2d1db1007a5d
Size: 23680
SHA256: 808c1eaf8266bef7039e0660b36d63f7143fc7872635c05b627b4c2f20b209ac
SHA1: 35ae5feec927adfff7ee9462bbbd73345ca2ced1
MD5sum: 54762cb2612805ab190dbc2c0e04f265
Description: ROS package library
         Library for retrieving information about ROS packages and stacks.
Homepage: http://wiki.ros.org/rospkg
```

Reply:
Doesn't the Debian rule apply to vanilla Debian too?

Reply:
Everything in this database assumes that you have the ROS sources available so ours packaged version will take precedence.

Reply:
It seems to be inconsistent with other `-modules` packages though: https://github.com/ros/rosdistro/blob/24ea867b61062bff1056961905f2c88210940bf0/rosdep/python.yaml#L4759